src/Utility/TensorFunction.h

```diff
@@ -56,6 +56,9 @@ class TensorFunc : public utl::SpatialFunction<Tensor>, public FunctionBase
 
   //! \brief Evaluates time derivatives of the function.
   Tensor timeDerivative(const Vec3& X) const;
+
+  //! \brief Evaluates second derivatives of the function.
+  utl::matrix4d<Real> hessian(const Vec3& X) const;
```

Comment:
here also, be consistent in the order of the declarations (gradient - hessian - timeDerivative, like elsewhere)

Reply:
ok